administration/src/bp-modules/cards/hooks/useCardGenerator.ts

```diff
@@ -59,6 +65,20 @@ const useCardGenerator = ({ region, initializeCards = true }: UseCardGeneratorPr
   const [createCardsMutation] = useCreateCardsMutation()
   const [deleteCardsMutation] = useDeleteCardsMutation()
   const appToaster = useAppToaster()
+  const [sendToFreinet] = useSendApplicationDataToFreinetMutation({
+    onCompleted: data => {
+      if (appToaster && data.result.errorMessage !== 'disabled for region') {
+        appToaster.show({
+          intent: data.result.success === true ? Intent.SUCCESS : Intent.DANGER,
+          message:
+            data.result.success === true
```

Comment:
🔧 i would just send a success message if `(data.result.success === true)` 
If its false (like `disabled for region`) nothing will be shown
In error cases just use `onError` function as mentioned above